integration-tests/infinispan-client/src/main/java/io/quarkus/it/infinispan/client/MarshallerConfiguration.java

```diff
@@ -33,6 +40,12 @@ FileDescriptorSource bookProtoDefinition() {
                 "  required int32 publicationYear = 3; // no native Date type available in Protobuf\n" +
                 "\n" +
                 "  repeated Author authors = 4;\n" +
+                "\n" +
+                "  enum Type {\n" +
```

Comment:
This has driven me totally crazy when working on the test. I did not realize about it, and this overrides the local protobuf file. So my enum definition did not exist